mass_mailing_custom_unsubscribe/migrations/8.0.2.0.0/pre-migrate.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
```

Comment:
Before migrating, they should be sure that have updated to latest 8.0 version, or we will need to maintain scripts for this version and the previous one in 2 branches.